code/modules/overmap/ships/computers/helm.dm

```diff
@@ -23,6 +23,8 @@ var/global/list/overmap_helm_computers
 	var/dy		//coordinates
 	var/speedlimit = 1/(20 SECONDS) //top speed for autopilot, 5
 	var/accellimit = 1 //manual limiter for acceleration
+	/// The mob currently operating the helm - The last one to click one of the movement buttons and be on the overmap screen. Set to `null` for autopilot or when the mob isn't in range.
+	var/mob/current_operator
```

Comment:
Would recommend using a weakref for this if you're adding new references.

Reply:
Pushed as a separate commit for review purposes, will squash once I know that's the route we're going.